src/state/index.ts

```diff
@@ -2,18 +2,45 @@ import { connectRouter, RouterState } from 'connected-react-router';
 import { createBrowserHistory, History } from 'history';
 import { combineReducers, createStore, Reducer } from 'redux';
 import { composeWithDevTools } from 'redux-devtools-extension';
+import * as uuid from 'uuid/v4';
 import { counterReducer, CounterState } from './counter/counter.reducer';
+import { homepageReducer, HomepageState } from './homepage/homepage.reducer';
 
 export interface AppState {
     counterState: CounterState;
+    homepageState: HomepageState;
     router: RouterState;
 }
 
 export const reducers = (history: History<any>): { [key in keyof AppState]: Reducer<any, any> } => ({
     counterState: counterReducer,
+    homepageState: homepageReducer,
     router: connectRouter(history),
 });
 
 export const history = createBrowserHistory({ basename: 'localsurvey' });
 
-export const store = createStore(combineReducers(reducers(history)), composeWithDevTools());
+// Define a preloadedState for createStore()
+// See: https://redux.js.org/recipes/structuring-reducers/initializing-state
+const preloadedState: any = {
+    homepageState: {
+        surveys: [
+            {
+                key: uuid.default(), // https://www.npmjs.com/package/uuid
+                name: 'Zoo visit questionnaire',
+                createdAt: 1549238929,
+                modifiedAt: 1549325329,
+                questionCount: 7,
+            },
+            {
+                key: uuid.default(),
```

Comment:
The `uuid` docs state that we should import a certain version (I'd use `v1` or `v4`)

It would look like this:
```
import uuid from 'uuid/v1'; // or 'uuid/v4'
```

Reply:
Yeah, but I'm using `@types/uuid`. Currently using v4 (random) btw